docs-source/content/userguide/experimental/istio.md

```diff
@@ -16,18 +16,14 @@ other components and services that have tracing enabled.
 
 #### Limitations
 
-The current experimental support for Istio has these limitations:
+The current support for Istio has these limitations:
 
 * It is tested with Istio 1.2.2 and later (up to 1.5), however it is tested with both single and
   multicluster installations of Istio.
-* Support is provided only for domains that are stored in persistent
-  volumes and created with the provided sample using the WLST option.
-  We intend to support Domain in Image and WDT options as well, but that is not currently
-  available.
 * Support is provided only for domains with a single dynamic cluster.
-  Multiple clusters and configured clusters are not currently supported.
+  Multiple clusters and configured clusters are not currently supported ???
```

Comment:
I think we can support this now ;).  It's a good test case. I'm pretty sure this statement is here because we only had the support built-in to the sample.